lua/mini/doc.lua

```diff
@@ -971,7 +971,17 @@ H.alias_replace = function(s)
       -- two values which might conflict if outputs are used as arguments.
       local name_escaped = vim.pesc(alias_name)
       local desc_escaped = vim.pesc(alias_desc)
-      s[i] = s[i]:gsub(name_escaped, desc_escaped)
+
+      if string.find(desc_escaped, "|", 1, true) then
+        desc_escaped = "(" .. desc_escaped .. ")"
+      end
+
+      local placeholder = "qqqq" .. name_escaped
```

Comment:
I don't think I like using this "placeholder" approach. It usually indicates that some logic is not quite right. My first instinct tells me that similar effect can be achieved at least by an extra `gsub` step or slightly more complicated Lua patterns.